MagickCore/option.c

```diff
@@ -1543,6 +1543,7 @@ static const OptionInfo
     { "Lanczos2Sharp", Lanczos2SharpFilter, UndefinedOptionFlag, MagickFalse },
     { "LanczosRadius", LanczosRadiusFilter, UndefinedOptionFlag, MagickFalse },
     { "LanczosSharp", LanczosSharpFilter, UndefinedOptionFlag, MagickFalse },
+    { "MagicSharp2013", MagicSharp2013Filter, UndefinedOptionFlag, MagickFalse },
```

Comment:
Not sure if we need the year?

Reply:
There are two variants of this algorithm (2013 and 2021), with different amounts of sharpening. I wanted to leave the possibility open that we can implement the other one as well.

Reply:
I saw that the author has also referred to these algorithms as "MagicKernelSharp" and "MagicKernelSharp+". I have renamed things accordingly.

Reply:
I read through the website and the paper again and realized that both my comments above are incorrect. I'm putting a very high-level summary of my current understanding here:

1. The author discovers the "Magic Kernel"
2. They notice that it introduces a slight blur so they include a sharpening filter (Sharp 2013 = {-1/4,3/2,-1/4}) to compensate. This is called "Magic Kernel Sharp 2013"
3. Instagram wants to add an extra "pop" to the image by sharpening it a little more. They change the sharpening filter to {-s,4+2s,-s}/4 for some value of s > 1. It's not explicitly mentioned what value of s Instagram uses. This filter is called "Magic Kernel Sharp+ 2013"
4. The author notices that "Sharp 2013" sharpens a little too much so they introduce a small blurring term {1,0,34,0,1}/36. Combining this term with "Sharp 2013" gives "Sharp 2021". This combined with the "Magic Kernel" gives "Magic Kernel Sharp 2021"
5. "Magic Kernel Sharp+ 2021" is a version of the above filter with a little extra sharpening. The exact parameters of this version are unclear.

I decided to implement "Magic Kernel Sharp 2021" in this PR because it does not blur or sharpen to 9 bits of accuracy and therefore can be considered the latest version of this algorithm.

I am unsure if it is worth implementing:
- the 2013 version (it is faster but not as accurate) OR
- the "pop" versions as their parameters are not specified

Please let me know your thoughts.

Reply:
I suggest both 2013 and 2021

Reply:
Would you prefer having "2013" and "2021" in the name or do you have other ideas?

Reply:
Done